release-notes/v3.28.1-release-notes.md

```diff
@@ -0,0 +1,31 @@
+30 Jul 2024
+
+#### Bug fixes
+
+ - Revert hcsshim library version bump from v0.11.6 back to v0.11.4 to fix Windows policies. [calico #9060](https://github.com/projectcalico/calico/pull/9060) (@coutinhop)
```

Comment:
maybe we could skip this note since the issue never made it to an actual release? (issue+revert happened in v3.28.1)

Reply:
Done!